Library/Homebrew/cli/args.rb

```diff
@@ -98,7 +98,8 @@ def context
 
       def only_formula_or_cask
         return :formula if formula? && !cask?
-        return :cask if cask? && !formula?
+
+        :cask if cask? && !formula?
```

Comment:
Nit: I wonder if an empty case statement would look better here.

```rb
case
when formula? && !cask?
  :formula
when cask? && !formula?
  :cask
end
```

There are always weird edge cases with guard statement cops. I prefer the original to be honest.

Reply:
That one would fail `Style/EmptyCaseCondition` but I've tweaked it now to use `if/elsif`.